ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -50,7 +50,7 @@ groups:
   jobs:
   - {{ build_test.name }}
   {%- for test in (tests) if not test.name=="StressNew" -%}
-    {%- for java_test_version in (java_test_versions) %}
+    {%- for java_test_version in (java_test_versions) if not java_test_version.name.endswith("JDK11") and not test.name.startswith("Windows") %}
```

Comment:
I almost hate to comment on one-line PRs, but if the intent is "only the things that act as blocking for `UpdatePassingRef`, then we should use the same as that block:

```
 - name: UpdatePassingRef	 - name: UpdatePassingRef
   public: true	   public: true
   serial: true	   serial: true
   plan:	   plan:
   - get: geode	   - get: geode
     passed:	     passed:
     {% for test in tests if not (test.name=="StressNew" or test.name.startswith("Windows")) -%}	     {% for test in tests if not (test.name=="StressNew" or test.name.startswith("Windows")) -%}
     - {{test.name}}TestOpenJDK8	     - {{test.name}}TestOpenJDK8
     {% endfor %}	     {% endfor %}
     trigger: true
```

and/or macro / explicitly define the "these are the jobs" list in our Jinja parameters somewhere.

Reply:
@pivotal-amurmann Should we then have an All or Complete so we can see them all?

Reply:
created a macro for all_gating_jobs and added "complete" tab